### PR TITLE
Add instructions for using bash completions

### PR DESCRIPTION
**Pull request by Wetrain**
This adds some instructions for enabling bash completions when using kind for a Bash shell.

### CODE REVIEW

**Comment by BenTheElder** [comment]
CLA still doesn't appear to be passing.

**Comment by Wetrain** [comment]
Hi sorry seems my email was wrong, i've corrected this now

**Comment by Wetrain** [comment]
Any update on getting this merged? :D

**Comment by BenTheElder** [comment]
I'd like to keep the readme high level and focused on the few quick things necessary to get an idea of what kind is, roughly how you'd use it, and why you might want to, as you can see we have a strong callout to the full docs at the top.

Ideally this would be a new guide under the user content, with zsh instructions.

File: README.md [hunk 1]
```diff
@@ -51,6 +51,23 @@ To use kind, you will need to [install docker].
 Once you have docker running you can create a cluster with `kind create cluster`  
 To delete your cluster use `kind delete cluster`
 
+### Auto completions
+
+To enable Bash completion for Kind commands, you can use the bash-completions project:
+
+```console
+sudo apt | dnf | pacman install bash-completion
```
Comment (BenTheElder) [inline README.md]: users are going to cut and paste this and run into it not being a real command, so we should split it out and instead point to the upstream install docs.

**Comment by Wetrain** [comment]
@BenTheElder Ok so this should be moved to the site repository with some instructions for each distro?

**Comment by BenTheElder** [comment]
yes please, a new page under the user guide.

sorry for the delay -- missed this but going back through the tracker now.

**Comment by BenTheElder** [comment]
let's file an updated PR to do this as a guide page under the user docs.

**Pull request closed without merging**